### PR TITLE
fixing flux tutorial section with JOBID

### PR DESCRIPTION
**Pull request by vsoch**
The cells are isolated, so we cannot use flux jobs and see anything from a previous submit. We also cannot use an environment variable to pipe in a JOBID. This fix will instead run flux job last directly in the command to properly show the job info resource spec and output. I also added showing the user how to write to a custom temporary file.

Ping @milroy this is a more commented version of this 
image

I want to get in individual fixes sooner than later in case we have live testers!

### CODE REVIEW

**Comment by vsoch** [comment]
Updated to have better output to show flux pstree. The current example shows a `.` which isn't super great. I also link to the actual nested examples. 
image

Here is what the output looks like (note the batch submit command isn't moved to the pstree block, I only did that testing if it might make the output better without `-a`, it did not!:

image

**Comment by vsoch** [comment]
okay I think I'm done with notebook tweaks.

**Comment by milroy** [comment]
Let's wait for feedback from the beta testers so we can put it in one PR.

File: notebook/flux.ipynb [hunk 1]
```diff
@@ -614,7 +614,7 @@
   },
   {
    "cell_type": "markdown",
-   "id": "42509fce",
+   "id": "13b7ffcb",
```
Comment (milroy) [inline notebook/flux.ipynb:690]: I can't leave a comment at the correct line, but line 668 and similar need a change:

```
"compute_jobreq.cwd = os.path.expanduser(\"~/flux-tutorial/flux-workflow-examples/job-submit-api/\") # set the CWD\n"
```
```

Comment (vsoch) [reply notebook/flux.ipynb:690]: Okay that line should be good - will look through our slack thread when I get back for more tweaks. https://github.com/rse-ops/flux-radiuss-tutorial-2023/pull/3/commits/86575eb03f64f7eff350aa0003469c0bc29872ba

File: notebook/flux.ipynb [hunk 2]
```diff
@@ -665,7 +665,7 @@
     "compute_jobreq = JobspecV1.from_command(\n",
     "    command=[\"./compute.py\", \"120\"], num_tasks=1, num_nodes=1, cores_per_task=1\n",
     ") # construct a jobspec\n",
-    "compute_jobreq.cwd = os.path.expanduser(\"..//flux-workflow-examples/job-submit-api/\") # set the CWD\n",
+    "compute_jobreq.cwd = os.path.expanduser(\"~/flux-tutorial/flux-workflow-examples/job-submit-api/\") # set the CWD\n",
```
Comment (milroy) [inline notebook/flux.ipynb:749]: Looks good, but there's one of these on lines 718 and 724, too.

File: notebook/flux.ipynb [hunk 3]
```diff
@@ -669,6 +750,31 @@
...
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ed65cb46-8d8a-41f0-bec1-92b9a89e6db2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# This is a new command to get info about your job from the id!\n",
+    "fluxjob = flux.job.submit(f,compute_jobreq)\n",
+    "fluxjobid = JobID(fluxjob.f58)\n",
+    "print(f\"🎉️ Hooray, we just submit {fluxjobid}!\")\n",
```
Comment (milroy) [inline notebook/flux.ipynb]: Nit, typo:
```suggestion
    "print(f\"🎉️ Hooray, we just submitted {fluxjobid}!\")\n",
```

**Review by milroy — CHANGES_REQUESTED** [review]
There's one path-related bug, but the rest is good. It looks like you incorporated everyone's feedback!

**Comment by vsoch** [comment]
okay all set: https://github.com/rse-ops/flux-radiuss-tutorial-2023/pull/3/commits/860528391bc1c4d42bdc9d60c20e58d569c1b3e1

I was the one that changed the expanduser to be ../ - I was thinking it could work with a relative path, but missed the detail of expanduser. It's probably safer to have it consistently reference home (`~`) and then use expanduser.

**Review by milroy — APPROVED** [review]
LGTM!

**Pull request merged**